src/engine/Api/V1/Parsers/Appointments.php

```diff
@@ -36,7 +36,8 @@ public function encode(array &$response)
             'customerId' => $response['id_users_customer'] !== NULL ? (int)$response['id_users_customer'] : NULL,
             'providerId' => $response['id_users_provider'] !== NULL ? (int)$response['id_users_provider'] : NULL,
             'serviceId' => $response['id_services'] !== NULL ? (int)$response['id_services'] : NULL,
-            'googleCalendarId' => $response['id_google_calendar'] !== NULL ? (int)$response['id_google_calendar'] : NULL
+            'googleCalendarId' => $response['id_google_calendar'] !== NULL ? (int)$response['id_google_calendar'] : NULL,
+            'platform' => $response['platform']
```

Comment:
add to v2, if v3 is available please add to v3
to keep this unchanged

Reply:
updated